src/main/java/com/stripe/model/Quote.java

```diff
@@ -940,193 +940,6 @@ public static class InvoiceSettings extends StripeObject {
     Long daysUntilDue;
   }
 
-  @Getter
-  @Setter
-  @EqualsAndHashCode(callSuper = false)
-  public static class QuotePhaseConfiguration extends StripeObject implements HasId {
```

Comment:
Double checking we haven't published a beta version with this embedded class right?

Reply:
Nope.